BlockzillaPackage/Sources/UIHelpers/UIHelpers.swift

```diff
@@ -0,0 +1,6 @@
+public struct UIHelpers {
+    public private(set) var text = "Hello, World!"
+
+    public init() {
+    }
+}
```

Comment:
These file should be part of the changes? Will be added something here? If this should be kept the header is missing.